packages/core/src/Assets/SvgComponents/onboarding/cfds.svg

```diff
@@ -1 +1,15 @@
\ No newline at end of file
+<svg width="85" height="112" viewBox="0 0 85 112" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
You'll need to minify all the svgs added.

Reply:
Minify with third party website? Because I did that now and I get like 0.1% smaller file